src/localizer.cpp

```diff
@@ -50,19 +50,27 @@ QString toUpper(const QLocale& locale, QString input) {
   return input.replace(0, 1, locale.toUpper(QString(input[0])));
 }
 
-QString toPascalCase(const QString& s) {
+QString changeCasing(const QString& s, bool shouldLowerCase) {
   QStringList words = s.split("_");
 
   for (int i = 0; i < words.size(); i++) {
     QString word = words.at(i);
     if (!word.isEmpty()) {
-      words[i] = word.at(0).toUpper() + word.mid(1);
+      if (shouldLowerCase) {
+        words[i] = word.at(0).toUpper() + word.mid(1).toLower();
+      } else {
+        words[i] = word.at(0).toUpper() + word.mid(1);
+      }
     }
   }
 
   return words.join("");
 }
 
+QString toPascalCase(const QString& s) { return changeCasing(s, true); }
```

Comment:
Might be helpful to add a comment for our future selves and any poor souls who come after us about when we expect to use toPascalCase vs toLanguageId.

Reply:
Good call, thanks. Added comments.